library/CHART-TEMPLATE/templates/deployment.yaml

```diff
@@ -38,6 +34,10 @@ spec:
       dnsConfig:
         {{- . | nindent 8 }}
       {{- end }}
+
+      hostNetwork: {{ include "common.hostNetwork.boolean" . }}
```

Comment:
I like the name to resolve the ambiguity!

As a nit, I'd rather use `bool` as it's the Go type name.

Reply:
Solved in [#cd27a22](https://github.com/newrelic/helm-charts/pull/756/commits/cd27a2262b8b953a7e5f921de89f8c81f3ead7c4)